src/app/components/OptimoPromos/README.md

```diff
@@ -0,0 +1,2 @@
+This are shared components that are 100% reusable.
+If a style is applied to the component make sure that style is applies to all Promos.
```

Comment:
Are these components all 100% reusable? If they are long term we'd want to move them up to: `/components`, keeping them seperate for now though is a good idea while we are experimenting. See WSTEAMA-110 for details of how we want to organise our components going forward.

Reply:
Technically yes? I believe the components are either very simple or complex but come from strong requirement fundamentals. 
100% reusable, yes, you might need to expand the styling, but the logic should be there!
Ofc I am talking in the context of Promos...